### PR TITLE
Mention dependency name for docker image tags

### LINKED ISSUE #24735: mention dependency-name for docker image tags
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore

### What part(s) of the article would you like to see updated?

The section about the `dependency-name` section talks about the format of a few dependency types, notably Java. I think a brief mention about the format for docker image tags would help developers attempting to use the `ignore` field in `dependabot.yml` on docker builds. I've mentioned an example [here](https://github.com/dependabot/dependabot-core/issues/6909#issuecomment-1487346279), but I think this information has persistent value.

Expected outcome is the `dependency-name` having a sentence or two to the effect of:

```txt
For docker image tags, the format is the full name of the repository; for example, given an image tag of <account ID>.dkr.ecr.us-west-2.amazonaws.com/base/foo/bar/ruby:3.1.0-focal-jemalloc, enter base/foo/bar/ruby.
```

### Additional information

I mentioned an example [here](https://github.com/dependabot/dependabot-core/issues/6909#issuecomment-1487346279) and [it was suggested that I open a PR to update the docs](﻿https://github.com/dependabot/dependabot-core/issues/6909#issuecomment-1487355986).

### PR DESCRIPTION
**Pull request by cvirtucio**
### Why:

Closes: #24735 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

The `dependency-name` section in [this subheader](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore) will look like this:

Screenshot 2023-03-28 at 2 46 50 PM

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Review by deivid-rodriguez — DISMISSED** [review]

**Comment by mchammer01** [comment]
Thanks for the technical review @deivid-rodriguez ✨ 
I'll review this on behalf of Docs now.

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ updates:
 
 Use the `allow` option to customize which dependencies are updated. This applies to both version and security updates. You can use the following options:
 
-- `dependency-name`—use to allow updates for dependencies with matching names, optionally using `*` to match zero or more characters. For Java dependencies, the format of the `dependency-name` attribute is: `groupId:artifactId`, for example: `org.kohsuke:github-api`.
+- `dependency-name`—use to allow updates for dependencies with matching names, optionally using `*` to match zero or more characters. For Java dependencies, the format of the `dependency-name` attribute is: `groupId:artifactId`, for example: `org.kohsuke:github-api`. For docker image tags, the format is the full name of the repository; for example, given an image tag of `<account ID>.dkr.ecr.us-west-2.amazonaws.com/base/foo/bar/ruby:3.1.0-focal-jemalloc`, enter `base/foo/bar/ruby`.
```
Comment (mchammer01) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:178]: Made some edits for improved readability, and to be in line with our content model.
```suggestion
- `dependency-name`—use to allow updates for dependencies with matching names, optionally using `*` to match zero or more characters. 
     - For Java dependencies, the format of the `dependency-name` attribute is: `groupId:artifactId`; for example: `org.kohsuke:github-api`. 
     - For Docker image tags, the format is the full name of the repository; for example, for an image tag of `<account ID>.dkr.ecr.us-west-2.amazonaws.com/base/foo/bar/ruby:3.1.0-focal-jemalloc`, use `base/foo/bar/ruby`.
```

**Review by mchammer01 — COMMENTED** [review]
@cvirtucio 👋🏻 - thanks for improving our docs ✨ 
I'll approve your PR once I've committed minor improvements that I've suggested from an editorial point of view 😃

**Review by mchammer01 — APPROVED** [review]
LGTM :shipit:

**Pull request merged**